### PR TITLE
UGDG help command for Search

### LINKED ISSUE #211: Update Search Command UG/DG

### PR DESCRIPTION
**Pull request by yuejunfeng0909**
resolves #211

### CODE REVIEW

**Review by haoyusimon — APPROVED** [review]
Looks good to merge.

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -290,6 +290,7 @@ ___
 |v2.0|user|create a shelf|store items into the shelf|
 |v2.0|user|remove a shelf|remove the shelf if the shelf is empty|
 |v2.0|user|view the estimated markup price for an item|know the percentage returns i can get from the markup|
+|v2.0|user|search for an item with very little information about it|find an
```
Comment (zh1huang) [inline docs/DeveloperGuide.md:293]: Seems incomplete?

**Review by zh1huang — CHANGES_REQUESTED** [review]
Good to merge except minor changes required

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yuejunfeng0909) [reply docs/DeveloperGuide.md:293]: Thanks, forgot to finish the sentence haha

**Pull request merged**